packages/dnb-eufemia/src/components/logo/Logo.tsx

```diff
@@ -111,6 +111,9 @@ function Logo(localProps: LogoProps) {
     return 'dnb'
   }, [brand, variant])
 
+  /** @deprecated Can remove this in v11 */
+  const height = !isNaN(parseFloat(size)) ? size : heightProp
```

Comment:
```suggestion
  const height = heightProp ?? size
```
I think, this would be more "readable"  `const height = parseFloat(size)>0 ? size : heightProp`, but the code suggestion above should be fine as well, I think. Because, if height is defined, we want to prioritize it. And if size is defined, it acts as a fallback for height. And if nothing is defined, its undefined, and will not be set.